apps/src/javalab/control-buttons.module.scss

```diff
@@ -0,0 +1,53 @@
+@import 'color.scss';
+
+@mixin controlButton {
+  float: left;
+  font-size: 15px;
+  width: 140px;
+  background-color: $orange;
+  border-color: $orange;
+  font-family: "Gotham 5r";
+  padding: 5px 12px;
+  margin: 5px 0 5px 5px;
+  justify-content: center;
+
+  &:hover {
+    color: $white;
+    box-shadow: none;
+  }
+}
+
+@mixin whiteControlButton {
+  @include controlButton;
+  background-color: $white;
+  border-color: $dark_charcoal;
+  color: $dark_charcoal;
+
+  &:hover {
+    color: $dark_charcoal;
+  }
+}
+
+.button {
```

Comment:
This may change depending on how @extend works out, but I think you can just @include controlButton at the top level, and then override it for the blue and white sub-classes (unless you want `.button` by itself to intentionally be unstyled? maybe to force a color selection?)